E    #       LOSS TOK2VEC  LOSS NER  ENTS_F  ENTS_P  ENTS_R  SCORE
---  ------  ------------  --------  ------  ------  ------  ------
wandb: Adding directory to artifact (.\corpus)... Traceback (most recent call last):
  File "C:\Users\astelzl\Miniconda3\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\astelzl\Miniconda3\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\astelzl\Miniconda3\lib\site-packages\spacy\__main__.py", line 4, in <module>
    setup_cli()
  File "C:\Users\astelzl\Miniconda3\lib\site-packages\spacy\cli\_util.py", line 69, in setup_cli
    command(prog_name=COMMAND)
  File "C:\Users\astelzl\Miniconda3\lib\site-packages\click\core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "C:\Users\astelzl\Miniconda3\lib\site-packages\click\core.py", line 782, in main
    rv = self.invoke(ctx)
  File "C:\Users\astelzl\Miniconda3\lib\site-packages\click\core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "C:\Users\astelzl\Miniconda3\lib\site-packages\click\core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "C:\Users\astelzl\Miniconda3\lib\site-packages\click\core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "C:\Users\astelzl\Miniconda3\lib\site-packages\typer\main.py", line 497, in wrapper
    return callback(**use_params)  # type: ignore
  File "C:\Users\astelzl\Miniconda3\lib\site-packages\spacy\cli\train.py", line 59, in train_cli
    train(nlp, output_path, use_gpu=use_gpu, stdout=sys.stdout, stderr=sys.stderr)
  File "C:\Users\astelzl\Miniconda3\lib\site-packages\spacy\training\loop.py", line 103, in train
    log_step, finalize_logger = train_logger(nlp, stdout, stderr)
  File "C:\Users\astelzl\Miniconda3\lib\site-packages\spacy\training\loggers.py", line 144, in setup_logger
    log_dir_artifact(path=log_dataset_dir, name="dataset", type="dataset")
  File "C:\Users\astelzl\Miniconda3\lib\site-packages\spacy\training\loggers.py", line 140, in log_dir_artifact
    dataset_artifact.add_dir(path, name=name)
  File "C:\Users\astelzl\Miniconda3\lib\site-packages\wandb\sdk\wandb_artifacts.py", line 413, in add_dir
    pool.map(add_manifest_file, paths)
  File "C:\Users\astelzl\Miniconda3\lib\multiprocessing\pool.py", line 364, in map
    return self._map_async(func, iterable, mapstar, chunksize).get()
  File "C:\Users\astelzl\Miniconda3\lib\multiprocessing\pool.py", line 771, in get
    raise self._value
  File "C:\Users\astelzl\Miniconda3\lib\multiprocessing\pool.py", line 125, in worker
    result = (True, func(*args, **kwds))
  File "C:\Users\astelzl\Miniconda3\lib\multiprocessing\pool.py", line 48, in mapstar
    return list(map(*args))
  File "C:\Users\astelzl\Miniconda3\lib\site-packages\wandb\sdk\wandb_artifacts.py", line 407, in add_manifest_file
    self._add_local_file(logical_path, physical_path)
  File "C:\Users\astelzl\Miniconda3\lib\site-packages\wandb\sdk\wandb_artifacts.py", line 683, in _add_local_file
    shutil.copyfile(path, f.name)
  File "C:\Users\astelzl\Miniconda3\lib\contextlib.py", line 120, in __exit__
    next(self.gen)
  File "C:\Users\astelzl\Miniconda3\lib\site-packages\wandb\sdk\interface\artifacts.py", line 932, in helper
    os.replace(tmp_file, path)
PermissionError: [WinError 5] Access is denied: 'C:\\Users\\astelzl\\.cache\\wandb\\artifacts\\obj\\md5\\4d\\tmp_7395D5A5' -> 'C:\\Users\\astelzl\\.cache\\wandb\\artifacts\\obj\\md5\\4d\\ed078c3ea52e6bce7760c4506debc6'